app/components/supplementary-file-browser/component.js

```diff
@@ -0,0 +1,132 @@
+import Ember from 'ember';
+import loadAll from 'ember-osf/utils/load-relationship';
+import config from 'ember-get-config';
+
+function fileDownloadPath(file, node) {
+    if (!file || !node) {
+        return;
+    }
+    if (file.get('guid')) {
+        return `${config.OSF.url}${file.get('guid')}/?action=download`;
+    }
+    return `${config.OSF.url}project/${node.get('id')}/files/osfstorage${file.get('path')}/?action=download`;
```

Comment:
path should be encoded with encodeURI or encodeURIComponent. I don't remember which one.